.all-contributorsrc

```diff
@@ -56,6 +56,13 @@
       "avatar_url": "https://avatars2.githubusercontent.com/u/3619527?v=3",
       "profile": "http://parvezk.github.io/",
       "contributions": []
+    },
+    {
```

Comment:
I think you'll need to [rebase](https://egghead.io/lessons/javascript-how-to-rebase-a-git-pull-request-branch) to get the latest changes. You have a merge conflict here.